### ISSUE TITLE
ES7 and jsx

### ISSUE THREAD

**Issue by pk81622** [touchstonejs/touchstonejs#107]
I'm new to programming so sorry if my issues are simple.
1. when I run the app, any jsx file is not recognized by the build
2. I imported some code and there is ES7 syntax (static, ::, @controllable, etc) and I'm getting errors. How do I get ES7 support?

**Comment by Duan112358** [comment]
touchstone is just a UI lib, ES7 is your coding choice, you should choose a complier, such as [babel](https://babeljs.io).

**Comment by pk81622** [comment]
I've been trying to figure out how to add that functionality to touchstone. I think I'm supposed to change something on the index.js file on touchstonejs-tasks? beyond that I'm not sure what to change. Any advice is greatly appreciated

**Comment by Duan112358** [comment]
yeah, you're right. The touchstonejs-tasks bind so tightly with the touchstone-starter project. What you should do is create an your own project, and import touchstone via npm, and compile use tools , such as [webpack](http://webpack.github.io) , and configure [babel-loader](https://github.com/babel/babel-loader) to support your ES7 features. There is [an awesome starter](https://github.com/petehunt/webpack-howto) for you.

**Comment by pk81622** [comment]
is there an easy way for me to modify the existing setup? Sorry for if my questions are silly.

**Comment by elliotatRetireup** [comment]
If you use the starter, you should be able to break it down looking at the views section. Most of it is pretty clear when you look at the app then the views. From there you can modify/create them to your liking. That is what I have been doing. 

Unfortunately it seems like the developer of this project is busy with other work, so I'm not sure if anything will be done to make this better or easier anytime soon.
